pantheon/src/main/java/tech/pegasys/pantheon/chainimport/internal/BlockData.java

```diff
@@ -28,8 +28,10 @@
 import java.util.stream.Stream;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+@JsonIgnoreProperties("comment")
```

Comment:
Would `@JsonIgnoreProperties(ignoreUnknown=true)` work here?  Or are you only explicitly wanting to allow the `comment` field?

Reply:
That would work, but I think it's better to explicitly ignore.  That way, it will be obvious if some optional field is misspelled or something.